src/elements/content-uploader/UploadsManagerAction.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { injectIntl, IntlShape } from 'react-intl';
+import { Button } from '@box/blueprint-web';
+
+import { RESIN_TAG_TARGET } from '../../common/variables';
+import { STATUS_ERROR } from '../../constants';
+
+import messages from '../common/messages';
+
+import './UploadsManagerAction.scss';
+
+export interface UploadsManagerActionProps {
+    hasMultipleFailedUploads: boolean;
+    intl: IntlShape;
+    onClick: Function;
```

Comment:
Same here. Can we make this more explicit

Reply:
Also should we consider extracting the types/interfaces out into their own file for this particular directory and just importing them? In our shared component library we saw that having the types in one place could be helpful. Just a thought.

Reply:
I think it is more reasonable for us to keep the interfaces in its own file because the usages of that prop would only be used in the component itself and its test file.